tfhe/src/shortint/parameters/mod.rs

```diff
@@ -787,7 +892,7 @@ pub const PARAM_MESSAGE_2_CARRY_0_KS_PBS: ClassicPBSParameters =
 pub const PARAM_MESSAGE_2_CARRY_1_KS_PBS: ClassicPBSParameters =
     PARAM_MESSAGE_2_CARRY_1_KS_PBS_GAUSSIAN_2M64;
 pub const PARAM_MESSAGE_2_CARRY_2_KS_PBS: ClassicPBSParameters =
-    PARAM_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64;
+    PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
```

Comment:
we should remove all params except 2_2 TUniform I believe, as here we have inconsistent kinds of parameters